doc/ideas/fr_technical_overview.md

```diff
@@ -0,0 +1,137 @@
+UI                 | UI17       | Looking for a SC2-Interface in AOE-style? UI-Modding should be made easy.
+UI                 | UI18       | Tooltips should be improved to properly showcase all the hidden information and boni for players who don't want to deeply research every facet of the game by themselves.
+UI                 | UI19       | Host of lobby to ban civilizations, give handicaps
+UI                 | UI20       | Idle villager button change colour when a villager goes idle
+AI                 | AI1        | Handles units more intelligent. Don't build in hazardous areas. Send military to clear out. Build more gates to prevent lock-ins. Better castle placement. Better Trade. Military units shouldn't fight in castle arrow fire. Units marching react and don't just go away. AI lames you (Barbarian). Understands how to do economic damage to your base. Adopts more to the current playing meta and also randomly choses to use older metas. AI can deliver villagers with transportships to other islands.
+Graphics           | GR1        | Zooming in and out with different settings: middle of screen, where mouse points, to selected units.
+Graphics           | GR2        | Units throw torches to put buildings on fire instead of fighting against them with their swords. Villagers have different attack animations depending on where they were working before attacking units/buildings, e.g. farms -> hay fork, lumberjacks -> axe, stone or gold -> pickaxe.
+Graphics           | GR3        | Animations for construction and destruction of buildings.
+Graphics           | GR4        | Unique military and economic unit skins for each different region. knights from Europe shouldn't look like the ones from Japan, but should be similar enough to tell they are both knights. At least 4 regions for unit graphics.
+Graphics           | GR5        | Beautiful weather effects (see G1)
+Graphics           | GR6        | Campaign cutscenes
+Graphics           | GR7        | Some graphical optimisations for competitive play by design, e.g. small trees, alignment grid, fish outlines, short walls, etc., other option: to redesign everything that the problems why those mods exists are fixed. e.g. make the look behind walls better by design -> no short walls needed.
+Graphics           | GR8        | Resource spots change size based on the amount of resource they have left.
+Graphics           | GR9        | Sync ram attack animation with the actual attack.
+Graphics           | GR10       | A special flag that raises on a building that is researching a technology, much like the garrison flag. (see GM44)
+Modding            | MD1        | Asset modification updates in-game models instantly. Live testing for modders. More challenging: Same for unit (stat) patches.
```

Comment:
> More challenging: Same for unit (stat) patches.

That's actually the most easiest thing to do. :D I wouldnt even mention it in here

Reply:
just copied it from the gameplay.md, someone wrote there "more challenging" :-D

Reply:
removed it